docs/sunfish.md

```diff
@@ -0,0 +1,93 @@
+# Installing for the first time and verified boot
+
+It is assumed that you have successfully built your image and signed it with
+your own keys.  Make sure that you know the location of the image and the AVB
+signing key.
```

Comment:
I don't believe anything here is sunfish-specific.   Perhaps rename this file to "installing.md", mention this is for Pixel devices and that it has been tested on sunfish.

Reply:
Also, a link to: https://source.android.com/setup/build/running could be useful as well.